docs/vision-usage.rst

```diff
@@ -163,52 +103,72 @@ locations if available.
     >>> landmarks[0].bounding_poly.vertices[0].y_coordinate
     162
 
+
 Logo Detection
 ~~~~~~~~~~~~~~
 
-Google Vision can also attempt to detect company and brand logos in images.
+With :meth:`detect_logos()`, you can identify brand logos in an image. Their
+shape and location in the image can be found by iterating through the detected
+logo's `vertices`.
 
 .. code-block:: python
 
     >>> from google.cloud import vision
     >>> client = vision.Client()
-    >>> image = client.image('./image.jpg')
+    >>> with open('./image.jpg', 'rb') as image_file:
+    ...     image = client.image(content=image_file.read())
     >>> logos = image.detect_logos(limit=1)
-    >>> results.logos[0].description
+    >>> first_logo = logos[0]
```

Comment:
Actually...it could be up to the limit or less.  I vaguely remember this varying slightly as I was playing with it.

If I set the limit to 10 and I knew my image had 8 logos in it, the I might only get back 3-4 logos that were detected.